Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -428,18 +433,17 @@ def prepare_ip_list_get_output(response: dict) -> str:
     hr_output = {
```

Comment:
This human-readable output also needs the IP range field.

Reply:
under the IP ranges will show the from_ip field, The HR column name would be IP range